src/components/StaticMetadata/StaticMetadata.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import renderWithTheme from '../../../tests/utils/withTheme';
+import StaticMetadata from './StaticMetadata';
+import { metadataList } from './__stubs__';
+
+describe('<StaticMetadata />', () => {
+	beforeEach(() => renderWithTheme(<StaticMetadata list={metadataList} />));
+
+	test('Renders with metadata', () => {
+		metadataList.forEach((metadata) => {
+			const labelColumn = screen.getByText(metadata.label);
+			expect(labelColumn).toBeInTheDocument();
+
+			if (typeof metadata.value === 'string') {
```

Comment:
As we only render string values for the POC I ommited the Tag render testcase. If there is a wise solution available to check in the test if metadata.value is a type of React.ReactNode andso test the existence of the Tag output, glad to hear :-)

Reply:
You can mock `<Tag />` and spy on the mock being called with the appropriate props. Arguably though you only have to mock the description list component and check for the props on that. Coverage on the internals of the description list exist in the component library already.

Reply:
Changed.